src/types/types.ts

```diff
@@ -222,6 +222,8 @@ interface DimensionContentStyling extends FontStyling {
   measureLabel: CellStyling;
 }
 
+export type MeasureTextStyling = HeaderStyling | MeasureContentStyling | DimensionContentStyling;
```

Comment:
Can the type not be `{ fontSize: string; fontFamily: string }`? That would make it easier when we move this hook to the shared util repo.

Or we might end up moving the measure text hook from sn-table into the util repo instead. But still, the goal should be that both repos should use the same measure hook.

Reply:
good point, I will have a look at sn-table so the typing and functionality match